### DISCUSSION TITLE
Inverse scroll setting ignored for Multimedia mouse pad

### DISCUSSION THREAD

**Discussion by geo064** [AppGround-io/bluetooth-keyboard-and-mouse-support#146] [Bugs & Problems]
--note to self:
created nov 2020
still not fixed dec 2021
--end

Inverse scroll setting is ignored for Multimedia mouse pad

To reproduce,
1. Enter Mouse/Keyboard mode
2. Open settings
3. Set Invert Scroll Direction toggle to ON
4. Try two-finger scroll to confirm setting is ON
5. Enter Multimedia mode
6. Try two-finger scroll, scrolling is NOT inverted

**Comment by Staini** [comment]
Custom layouts are also affected.
An App-wide invert scroll setting would be nice

Update: Oh, and the setting for touchpad clicks isn't App-wide, too.

**Comment by AppGround-io** [comment]
Thank you both for reporting this. 
The initial idea was to have the possibility to change the touch pad settings based on the mode. In the next version of the app the global settings will be used because there is currently no interface for that and it may not make sense for these values to change based on the mode.

**Reply by Staini** [reply]
Thank you very much!
